demisto_sdk/commands/doc_reviewer/tests/doc_reviewer_test.py

```diff
@@ -1,6 +1,90 @@
 from typing import List
 
+import pytest
+
 from demisto_sdk.commands.doc_reviewer.doc_reviewer import DocReviewer
+from TestSuite.json_based import JSONBased
+from TestSuite.yml import YAML
+
+
+@pytest.fixture()
+def malformed_integration_yml(integration) -> YAML:
+    integration.yml.write("1: 2\n//")
+    return integration.yml
+
+
+@pytest.fixture()
+def malformed_incident_field(pack) -> JSONBased:
+    incident_field = pack.create_incident_field("malformed")
+    incident_field.write_as_text("{\n '1': '1'")
+    return incident_field
```

Comment:
Why not move it to `demisto-sdk/demisto_sdk/tests/test_tools.py` and import it from there for both test files? 
Code duplication should be minimalized

Reply:
it can be done, but i wasn't sure if importing it from the another test module would be good, ill do that

Reply:
anyway it must be added with # noqa - for example - 
```from demisto_sdk.commands.common.tests.tools_test import (  # noqa F401
    malformed_incident_field, malformed_integration_yml
)```